National Library of Poland ISBN.js

```diff
@@ -0,0 +1,118 @@
+{
+	"translatorID": "aa7f310e-10d3-4209-91dc-88301e7070c6",
+	"label": "National Library of Poland ISBN",
+	"creator": "Maciej Nux Jaros",
+	"target": "",
+	"minVersion": "4.0",
+	"maxVersion": "",
+	"priority": 97,
+	"inRepository": true,
+	"translatorType": 8,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2023-05-20 15:41:08"
+};
```

Comment:
that's the ; the linter is complaining about

Reply:
Should be fine now. That came from editing in VSC.